project/make/.integration-daemon-start

```diff
@@ -16,6 +16,7 @@ export DOCKER_GRAPHDRIVER=${DOCKER_GRAPHDRIVER:-vfs}
 export DOCKER_EXECDRIVER=${DOCKER_EXECDRIVER:-native}
 
 if [ -z "$DOCKER_TEST_HOST" ]; then
+	unset DOCKER_HOST
```

Comment:
Well, tests also rely on unix://var/run/docker.sock, do you mean we should always have a DOCKER_HOST to run integration-cli tests, even though it's unix://.../docker.sock?